package.json

```diff
@@ -15,7 +15,7 @@
 	"scripts": {
 		"build": "rm -rf app/js && tsc",
 		"serve": "npx serve -L ./app",
-		"vendor": "npm ci --ignore-scripts && npm --prefix ./build ci --ignore-scripts && npm --prefix ./build run vendor",
-		"styles": "npm --prefix ./twcss ci --ignore-scripts && npm --prefix ./twcss run styles"
+		"vendor": "npm exec --prefix ./build vendor",
```

Comment:
i've combined the vendor script with build script since it's necessary to run the app anyways and `npm install` (or `npm ci`) should have been ran by then as well